frontend/src/scenes/retention/queries.ts

```diff
@@ -4,7 +4,7 @@ import { performQuery } from '~/queries/query'
 import { ActorsQuery, NodeKind, RetentionQuery } from '~/queries/schema'
 
 export function retentionToActorsQuery(query: RetentionQuery, selectedInterval: number, offset = 0): ActorsQuery {
-    const group = query.aggregation_group_type_index !== undefined
+    const group = query.aggregation_group_type_index != null
```

Comment:
This is where the issue is. A `null` would previously satisfy the condition and create a group-based actors query instead of the correct person-based one.